services-api/src/main/java/io/scalecube/services/MicroservicesContext.java

```diff
@@ -0,0 +1,16 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+public interface MicroservicesContext {
+
+  String id();
+
+  ServiceCall call();
+
+  Address serviceAddress();
+
+  ServiceDiscovery discovery();
```

Comment:
Let's rename it this legacy `.discovery()` to `.serviceDiscovery()`.  @ronenhamias wdyt?